ci/test-conformance-eks.sh

```diff
@@ -71,6 +72,10 @@ case $key in
     KUBECONFIG_PATH="$2"
     shift 2
     ;;
+    --k8s-version)
```

Comment:
are you using this somewhere in Jenkins? I don't see a reference to it in the yaml files.

BTW, the idea was to default to the value included in https://github.com/vmware-tanzu/antrea/blob/master/ci/k8s-conformance-image-version, but maybe this version doesn't work for EKS because they use an older version of K8s for their clusters?

Reply:
This won't work as I just tested the highest supported version as of now is still 1.15. However wanted to expose this config so we can edit Jenkins yamls or let user specify this without having to modify the script.

Reply:
My bad. I thought this was the version of the conformance tests, not the K8s version itself.